exp/api/v1alpha3/conversion_test.go

```diff
@@ -36,22 +36,4 @@ func TestFuzzyConversion(t *testing.T) {
 		Hub:    &infrav1exp.AzureMachinePool{},
 		Spoke:  &AzureMachinePool{},
 	}))
-
-	t.Run("for AzureManagedCluster", utilconversion.FuzzTestFunc(utilconversion.FuzzTestFuncInput{
```

Comment:
I opted to skip the pre-v1beta1 conversions because AKS was never a thing for pre-v1beta1 in the non-experimental surface area. Thoughts @CecileRobertMichon ?

Reply:
Since exp is just a feature flag + location in the codebase, I'm not sure I'm following why that should affect conversion. What happens if a user has an existing v1alpha4 AKS cluster (with exp feature flag on) and now that we're graduating they decide to upgrade to v1beta1? the conversion would still occur so the tests are valuable, v0.4.x to v1.8.0 is a valid upgrade path.

Reply:
Sorry, I spaced out entirely on what I actually did. Of course I didn't skip pre-v1beta1. This was simply a test migration oversight, nicely identified @nojnhuh!

Time for some lunch.